index.html

```diff
@@ -1,92 +1,106 @@
       people out. I often answer questions related to languages or libraries I
       have never used, because the answers are (in a programmer kind of way)
-      common sense.
+      common sense.-->
     </p>
     <p>
-      Alternatively, it can be seen as..
+      anu, ni li ken e sona e ni:
+      <!--Alternatively, it can be seen as..-->
     </p>
     <blockquote>
       <span class="name">Foobar123:</span>
       <p class="message">
-        I have a question about Java but I'm too lazy to actually formalize it
+        mi jo e sona wile Java taso, mi wile ala e toki e ni tan, jan jo sona la lon ala li ma ni.
+        <!--I have a question about Java but I'm too lazy to actually formalize it
         in words unless there's someone on the channel who might be able to
-        answer it
+        answer it-->
       </p>
     </blockquote>
     <p>
-      ..which is just lazy. If you're not willing to do the work to solve your
-      problem, why should we?
+      ni li wile ala pali kin. sina wile ala e pona e nimi linluwi sina tan, jan ete li wile tan seme?
+      <!--..which is just lazy. If you're not willing to do the work to solve your
+      problem, why should we?-->
     </p>
     <p>
-      The solution is not to ask to ask, but just to ask. Someone who is idling
+      nasin li ni: o toki alasa ala tan toki alasa a! sina toki alasa e toki pi pakala la, jan seme li lon e toki linluwi li wile e pana sona.
+      <!--The solution is not to ask to ask, but just to ask. Someone who is idling
       on the channel and only every now and then glances what's going on is
       unlikely to answer to your "asking to ask" question, but your actual
-      problem description may pique their interest and get them to answer.
+      problem description may pique their interest and get them to answer.-->
     </p>
     <p>
-      So, to summarize, don't ask
-      <em>"Any Java experts around?"</em>,
-      but rather ask
-      <em>"How do I do [problem] with Java and [other relevant info]?"</em>
+      a, sina o wile ala toki alasa e ni:
+      <!--So, to summarize, don't ask-->
+      <em>"jan sona Java li lon anu seme?"</em>,
+      <!--<em>"Any Java experts around?"</em>,-->
+      taso, toki e ni:
+      <!--but rather ask-->
+      <!--<em>"How do I do [problem] with Java and [other relevant info]?"</em>-->
+      <em>"pali pona e [pakala] Java li nasin seme? sona ete e ni: [sona]."</em>
     </p>
     <p>
-      Other similar problems: <a href="https://xyproblem.info/">The XY Problem</a>, <a href="https://nohello.net/">No Hello</a>.
-      Further reading: <a href="https://stackoverflow.com/help/how-to-ask">How do I ask a good question?</a>,
-      or if you have more time: <a href="http://catb.org/~esr/faqs/smart-questions.html">How To Ask Questions The Smart Way</a>.
+      <!--Other similar problems--> pakala sama ete: <a href="https://xyproblem.info/">pakala XY</a>, <a href="https://nohello.net/">toki! (ala)</a>.
```

Comment:
`ante` is a word that exists that probably solves most, if not all, of the uses of `ete`

```suggestion
      <!--Other similar problems--> pakala sama ante: <a href="https://xyproblem.info/">pakala XY</a>, <a href="https://nohello.net/">toki! (ala)</a>.
```